### ISSUE TITLE
Add content on the ASG with MIG workshop to reference the architecture component being configured

### ISSUE THREAD

**Issue by ivallhon** [awslabs/ec2-spot-workshops#137]
The workshop assumes that the attendee is following in his mind the architectural diagram presented on the [introduction](https://ec2spotworkshops.com/ec2-auto-scaling-with-multiple-instance-types-and-purchase-options/architecture.html). It would be good to add some extra context on the different steps highlighting what's being configured so it gives a big picture understanding.

**Comment by ruecarlo** [comment]
Was this part of the last PR for ASGs ? If so, can this be closed ?

**Comment by ivallhon** [comment]
No, this wasn't part of #136 , this is feedback I got while doing a dry-run of the updated workshop and that I'll look to incorporate.
